modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -95,13 +101,39 @@ pullSecret: '<pull_secret>' <6>
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
 ====
 
-. Complete the steps in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation to download {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory. You need the {oci-ccm} manifests for deploying the {oci-ccm} during cluster installation so that {product-title} can connect to the external {oci} platform. You need the {oci-csi} custom manifests for deploying the {oci-csi} driver during cluster installation so that {product-title} can claim required objects from {oci}.
+. If you used a stack to provision OCI infrastructure resources: Copy and paste the `dynamic_custom_manifest` output of the OCI stack into a file titled `manifest.yaml` and save the file in the `openshift` directory.
 
-. Access the custom manifest files that are provided in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation.
+. If you did not use a stack to provision OCI infrastructure resources: Download and prepare custom manifests for Oracle.
+
+.. Go to link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files[Configuration Files] (Oracle documentation).
+
+.. Download the {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory.
```

Comment:
Currently, there are 8 files under the Oracle repo that need to be placed in the OpenShift directory. The user needs to edit the `01-oci-driver-configs.yml` file to update the compartment OCID, VCN OCID, subnet OCID from the load balancer and security lists OCID.

I would suggest getting in touch with Oracle folks for more information about it.

Reply:
I can ask an Oracle engineer for feedback on making this overall part more accurate